pkg/webhook/useridentity/mutatingWebhook.go

```diff
@@ -92,6 +95,17 @@ func (a *AdmissionHandler) mutateResource(ar v1.AdmissionReview) *v1.AdmissionRe
 	}
 
 	annotations := obj.GetAnnotations()
+
+	// Do not change the userInfo if the object is being created by the AppMgr.
+	// This value is stamped onto the Subscription object by Appmgr (Only for subscriptions)
+	klog.V(2).Infof("Object kind:%s, User request:%s, expected:%s", obj.GetKind(), ar.Request.UserInfo.Username, UserIDforAppMgr)
+	if obj.GetKind() == "Subscription" && ar.Request.UserInfo.Username == UserIDforAppMgr {
```

Comment:
I would rather have a flag to specify which identity should be skipped, and has UserIDforAppMgr as the default value.

Reply:
Quijing, thanks for the review. This is all new to me. Can you give me more details about this flag? Are you asking this flag to be passed in the AdmissionRequest?

Reply:
https://github.com/open-cluster-management/multicloud-operators-foundation/blob/main/cmd/webhook/app/options/options.go#L33 add a flag entry here

Reply:
Quijian, thank you for the hint. Hopefully I got it right. Please review. Thanks.